parsely/parselyandroid/ParselyTracker.java

```diff
@@ -205,16 +337,18 @@ private void persistQueue(){
         try{
             FileInputStream fis = this.context.getApplicationContext().openFileInput(
                     this.storageKey);
-        ObjectInputStream ois = new ObjectInputStream(fis);
-        //noinspection unchecked
-        storedQueue = (ArrayList<Map<String, Object>>)ois.readObject();
-        ois.close();
-        } catch(EOFException ex){
+            ObjectInputStream ois = new ObjectInputStream(fis);
+            //noinspection unchecked
+            storedQueue = (ArrayList<Map<String, Object>>)ois.readObject();
+            ois.close();
+        } catch(EOFException ex) {
             PLog("");
-        }
-        catch(Exception ex){
+        } catch (FileNotFoundException ex) {
+            // Nothing to do here. Means there was no saved queue.
+        } catch(Exception ex) {
```

Comment:
I don't know why it's there. The FileNotFound was the only exception I hit, but yes, crashing an app because of something unforeseen here seems unnecessary.